content/concepts/the-rippled-server/rippled-server-modes.md

```diff
@@ -2,61 +2,83 @@
 
 The `rippled` server software can run in several modes depending on its configuration, including:
 
-* Stock server - follows the network with a local copy of the ledger.
-* Validating server, or _validator_ for short - participates in consensus (and does everything a stock server does, too).
-* `rippled` server in stand-alone mode - for testing. Does not communicate to other `rippled` servers.
+- [**Network Mode**](#network-mode) - This is the main mode of the server: it follows the peer-to-peer network, processes transactions, and maintains some amount of [ledger history](ledger-history.html). This mode can be configured to do any or all of the following roles:
+    - [**Validator**](#validators) - Helps secure the network by participating in consensus
+    - [**API Server**](#api-servers) - Provides [API access](get-started-with-the-rippled-api.html) to read data from the shared ledger, submit transactions, and watch activity in the ledger. Optionally, this can be a [**Full History Server**](#full-history-servers), which keeps a complete record of transaction and ledger history.
+    - [**Hub Server**](#public-hubs) - Relays messages between many other members of the peer-to-peer network.
+- [**Reporting mode**](#reporting-mode) - A specialized mode for serving API requests from a relational database. Since it does not participate in the peer-to-peer network, it requires a connection to a Network Mode server. [New in: rippled 1.7.0][]
```

Comment:
```suggestion
- [**Reporting mode**](#reporting-mode) - A specialized mode for serving API requests from a relational database. Since it does not participate in the peer-to-peer network, it requires a connection to a Network Node server. [New in: rippled 1.7.0][]
```